guides/developer/service-overviews/orca.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: single
+title:  "Orca Internals Overview"
+sidebar:
+  nav: guides
+---
+
+Spinnaker is known and often loved for enabling accessible, highly configurable and composable workflows through its pipelines and tasks. This is largely thanks to the code that makes up Orca.
+
+From Orca's [README](https://github.com/spinnaker/orca/blob/master/README.md):
+
+> Orca is the orchestration engine for Spinnaker. It is responsible for taking a pipeline or task definition and managing the stages and tasks, coordinating the other Spinnaker services.
+
+Orca is a stateless, horizontally scalable service. Spinnaker's execution state is persisted to a backend store and work is distributed evenly through a work queue. So long as your backend persistence layer has capacity, you can freely add or remove Orca servers to match workload demands.
+
+# Internals
+
+## Domain
+
+The primary domain model is an **Execution**, of which there are two types: `PIPELINE` and `ORCHESTRATION`. The `PIPELINE` type is for pipelines while `ORCHESTRATION` is unscheduled, ad-hoc API-submitted actions and what you see in the "Tasks" tab in an Application view. At this point in Spinnaker’s life, these two types are nearly programmatically identical, the key difference being that a Pipeline has a predefined persistent configuration, whereas an Orchestration is arbitarily configured at request-time and serial in execution. 
```

Comment:
"ORCHESTRATION is unscheduled" feels wrong to me - I'd switch to either "`ORCHESTRATION`s are unscheduled" or "the `ORCHESTRATION` type is for unscheduled ... "

Reply:
Could mention that you're talking about the UI, since it might not be totally clear - "and what you see in the UI's "Tasks" tab in an Application view"